modes/basic-viewer/package.json

```diff
@@ -1,43 +0,0 @@
-{
```

Comment:
Has it been discussed that the basic viewer mode be removed? I assume so, just checking.